### PR TITLE
Add effect for Darkened Sight and unify effects for Traveler's Transit

### PR DESCRIPTION
**Pull request by DocSchlock**
Adds a new effect for Darkened Sight.

Also adds a single consolidated ChoiceSet effect for Traveler's Transit.

### CODE REVIEW

File: packs/spell-effects/spell-effect-travelers-transit.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "Climb",
```
Comment (simonward) [inline packs/spell-effects/spell-effect-travelers-transit.json:27]:
```suggestion
                        "label": "PF2E.Actor.Speed.Type.Climb",
```

File: packs/spell-effects/spell-effect-travelers-transit.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        "value": "climb"
+                    },
+                    {
+                        "label": "Swim",
```
Comment (simonward) [inline packs/spell-effects/spell-effect-travelers-transit.json:31]:
```suggestion
                        "label": "PF2E.Actor.Speed.Type.Swim",
```

File: packs/spell-effects/spell-effect-travelers-transit.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        "value": "swim"
+                    },
+                    {
+                        "label": "Fly",
```
Comment (simonward) [inline packs/spell-effects/spell-effect-travelers-transit.json:35]:
```suggestion
                        "label": "PF2E.Actor.Speed.Type.Fly",
```

**Comment by simonward** [comment]
We may want to change the icon for Darkened Sight (and effect) as it currently shares the same icon as Hidden.

**Comment by DocSchlock** [comment]
I'll try to find another good icon in the system.

**Pull request merged**